Add 4th Round Diligence Audit Report (#23)
Consensys/linea-contracts: README.md

# linea-contracts

## <span style="color: yellow;">Pending deprecation notice</span>

> This repository will soon be deprecated and the Linea smart contracts will be available at the [Linea Monorepo](https://github.com/Consensys/linea-monorepo/)

Reviewer: ```suggestion
This repository is deprecated and the Linea smart-contracts are available at the [Linea Monorepo](https://github.com/Consensys/linea-monorepo/)
```
